src/test/mir-opt/inline/inline-closure-borrows-arg.rs

```diff
@@ -7,53 +7,11 @@ fn main() {
     println!("{}", foo(0, &14));
 }
 
+// EMIT_MIR rustc.foo.Inline.after.mir
```

Comment:
Okay looks like inline tests all have this problem, so I'm leaning in favor of a separate PR.